.github/workflows/benchmark.yml

```diff
@@ -0,0 +1,199 @@
+name: Benchmark Pull Request
+on:
+  pull_request:
+  repository_dispatch:
+    types: [ reporters-db-pr ]
+
+env:
+  main: "$(/usr/bin/git log -1 --format='%H')"
+
+jobs:
+  benchmark:
+    name: PR comment
+    if: github.event_name == 'pull_request'
+    runs-on: ubuntu-latest
+    steps:
+      #----------------------------------------------
+      #       check-out repo and set-up python
+      #----------------------------------------------
+      - name: Check out repository
+        uses: actions/checkout@v2
+      - name: Set up python
+        id: setup-python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
+      #----------------------------------------------
+      #  -----  install & configure poetry  -----
+      #----------------------------------------------
+      - name: Install Poetry
+        uses: snok/install-poetry@v1
+        with:
+          virtualenvs-create: true
+          virtualenvs-in-project: true
+          installer-parallel: true
+          version: 1.2.0rc2
```

Comment:
We're doing something weird here, so it needs an explanation. That way, in the future, we'll know that we can stop using the `rc` version.

Reply:
Im testing `installer-parallel: true` I think it can be dropped and was/is for matrices